bower.json

```diff
@@ -15,17 +15,19 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-datetime": "^0.9.0",
-    "purescript-foreign": "^0.7.0",
-    "purescript-node-buffer": "^0.2.0",
-    "purescript-node-path": "^0.4.0",
-    "purescript-unsafe-coerce": "^0.1.0",
-    "purescript-nullable": "~0.2.1",
-    "purescript-node-streams": "~0.4.0",
-    "purescript-exceptions": "~0.3.3"
+    "purescript-datetime": "^1.0.0",
+    "purescript-foreign": "^1.0.0",
+    "purescript-node-buffer": "^1.0.0",
+    "purescript-node-path": "^1.0.0",
+    "purescript-unsafe-coerce": "^1.0.0",
+    "purescript-nullable": "~1.0.0",
+    "purescript-node-streams": "~1.0.0",
+    "purescript-exceptions": "~1.0.0",
+    "purescript-js-date": "^1.0.0",
+    "purescript-globals": "^1.0.0"
   },
   "devDependencies": {
-    "purescript-console": "~0.1.1"
+    "purescript-console": "~1.0.0"
```

Comment:
Yeah, it's a mixture of: bower used to default to `~`, and it didn't matter as we were on `0.x.y` anyway. As of the v1 releases using `^` means we can make additive changes to libraries without having to re-release things further down the chain.